src/features/expressions/shared-tests/functions/externalApi/should-return-empty-string-on-undefined-result.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "Should return empty string on undefined result",
+  "expression": ["externalApi", "testId", "results[0].id"],
+  "expects": "",
```

Comment:
I prefer null in this case as well, which is more consistent with the rest of the expression language